src/api/EscargotPublic.h

```diff
@@ -1181,6 +1183,8 @@ class ESCARGOT_EXPORT ObjectRef : public PointerValueRef {
 
     void enumerateObjectOwnProperies(ExecutionStateRef* state, const std::function<bool(ExecutionStateRef* state, ValueRef* propertyName, bool isWritable, bool isEnumerable, bool isConfigurable)>& cb);
 
+    uint64_t length(ExecutionStateRef* state);
```

Comment:
IMO we can comment to user what value should be returned by this function
ex) undefined-> ?
     nan-> ?

Reply:
Thanks for the review. I added some comment for it.